styles/RadioButtons.module.css

```diff
@@ -22,11 +22,12 @@
   -moz-appearance: none;
 }
 .button:focus ~ .label {
-  border-style: dashed !important;
+  outline: rgb(124, 186, 231) 2px solid;
```

Comment:
<img width="708" alt="image" src="https://user-images.githubusercontent.com/86619099/233370094-a197f154-b04d-45ca-8642-d80baba47a5d.png">
Is this intentional? I think it's a bit funky

Reply:
I think coloring the outline is more standard than adding a second layer